src/components/toolTip/ToolTip.module.css

```diff
@@ -0,0 +1,38 @@
+.wrapper {
+  display: inline-block;
+  position: relative;
+}
+
+.content {
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: var(--web-app-size-xs);
+  color: var(--web-app-color-white);
+  background: var(--web-app-color-black);
+  font-size: var(--web-app-font-size-sm);
+  font-family: var(--web-app-content-font-family);
+  line-height: var(--web-app-line-height-xs);
+  white-space: nowrap;
```

Comment:
All the design tokens in this file need to be changed to --conduction design tokens. 

The reason behind this is reusability, the --web-app design tokens will work perfectly in the Open Catalogi project, but this will break if we use it in a different project.